omod/src/main/webapp/resources/scripts/diagnoses/diagnoses-angular.js

```diff
@@ -4,7 +4,7 @@ var app = angular.module('diagnoses', [])
             // I don't know how to use an angular template programmatically, so use an underscore template instead. :-(
             var itemFormatter = _.template($('#' + attrs.itemformatter).html());
             element.autocomplete({
-                source: emr.fragmentActionLink("coreapps", "diagnoses", "search"),
+                source: emr.fragmentActionLink("coreapps", "diagnoses", "search", {diagnosisSets: attrs.diagnosissets}),
```

Comment:
When you test, just make sure you test that when diagnosisSets is null, this still operates correctly (ie doesn't break existing functionality).

Reply:
Ok, this is handled at [form creation](https://github.com/openmrs/openmrs-module-coreapps/pull/199/files#diff-e0e5822659198b9ceada75532b7be803R78) for the form and [back-end here](https://github.com/openmrs/openmrs-module-coreapps/pull/198/commits/6772566d8abb1378b01197f0d9d4f5bd14e1d7b8#diff-15549579e8c094656a9104bb0e207449R59) .

Reply:
Cool thanks @Ruhanga, just wanted to make sure you tested...